src/xdocs/checks/whitespace/whitespaceafter.xml

```diff
@@ -122,88 +122,96 @@
       </subsection>
 
       <subsection name="Examples" id="Examples">
-        <p>
+        <p id="Example1-config">
           To configure the check:
         </p>
         <source>
-&lt;module name="WhitespaceAfter"/&gt;
+&lt;module name=&quot;Checker&quot;&gt;
+  &lt;module name=&quot;TreeWalker&quot;&gt;
+    &lt;module name=&quot;WhitespaceAfter&quot;/&gt;
+  &lt;/module&gt;
+&lt;/module&gt;
         </source>
-        <p>Example:</p>
-        <div class="wrapper">
-          <pre>
-public void myTest() {
-    if (foo) {              // OK
-        //...
-    } else if(bar) {        // violation
-        //...
+        <source>
+class Example1 {
+  void example() throws Exception {
+    if (true) {
+    } else if(false) { // violation 'not followed by whitespace'
     }
 
-    testMethod(foo, bar);   // OK
-    testMethod(foo,bar);    // violation
+    testOne(&quot;x&quot;, &quot;y&quot;);
+    testOne(&quot;z&quot;,&quot;o&quot;); // violation 'not followed by whitespace'
 
-    for (;;){}               // OK
-    for(;;){}                // violation, space after 'for' is required
+    for (int i = 0; i &lt; 10; i++){}
```

Comment:
Is this compile error on `for(;;){}` ?

Reply:
Restored. I don't remember why I changed it. It compiles so it should be fine.